modules/nextflow/src/main/groovy/nextflow/extension/ChannelFactoryInstance.groovy

```diff
@@ -71,7 +71,7 @@ class ChannelFactoryInstance implements ChannelFactory {
     @Override
     Object invokeExtensionMethod(String method, Object[] args) {
         def result = invoke0(method,args)
-        NodeMarker.addSourceNode("channel.${this.scope}.${method}", result)
+        NodeMarker.addSourceNode(scope ? "channel.${this.scope}.${method}" : "channel.${method}", result)
```

Comment:
Think we can remove completely the support for `scope`

Reply:
I think is required to be compatible with current scripts who doesn't use the new syntax

Reply:
think we can go with a breaking change here